docs/cli/built-in-commands/addons.md

```diff
@@ -0,0 +1,68 @@
+# Add-on Management
+
+Set of CLI commands that allows listing, installing, updating and uninstalling add-ons.
+
+[TOC]
+
+## `addons:list`
+
+Lists all add-ons available in the system.
+
+    php eecli.php addons:list
+
+List add-ons that can be installed:
```

Comment:
@matthewjohns0n is this correct? 

List installed lists add-ons that can be installed?  Not ones that are installed?

Reply:
I see what you're saying - yes this needs to change. Good catch.